qt/python/instrumentview/instrumentview/Peaks/test/test_detector_peaks.py

```
@@ -79,16 +79,16 @@ def test_get_x_values_and_labels_empty_when_no_matching_spectrum_numbers(self):
-    def test_get_positions_and_labels_peak_id_larger_than_all_detector_ids(self):
+    def test_get_peaks_indices_and_labels_peak_id_larger_than_all_detector_ids(self):
         # Peak detector ID (999) is larger than every entry in detector_ids.
         # np.searchsorted would return an out-of-bounds index; the valid mask must
         # filter it out so the result is empty rather than raising an IndexError.
         peak = Peak(999, 0, (1, 0, 0), 100, 10, 10, 10)
         wdp = self._create_workspace_detector_peaks([DetectorPeaks([peak])])
         detector_positions = np.array([[1, 1, 1], [2, 2, 2]])
         detector_ids = np.array([1, 2])
-        positions, labels = wdp.get_positions_and_labels(detector_positions, detector_ids)
-        self.assertEqual(0, len(positions))
+        indices, labels = wdp.get_peaks_indices_and_labels(detector_positions, detector_ids)
```

Ruff (0.16.1)

[warning] Unpacked variable `labels` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)